### PR TITLE
V4 tutorials

### LINKED ISSUE #912: Update "Market" tutorial section
As the V4 Market UI looks different from the V3 version, we'll need to update the [publish tutorial](https://deploy-preview-819--docs-oceanprotocol.netlify.app/tutorials/marketplace-publish-data-asset/).
@akshay-ap Will give you everything needed once we have a stable version of Market V4

cc @kremalicious

### PR DESCRIPTION
**Pull request by akshay-ap**
Fixes #912

Changes proposed in this PR:

- [x] V4 Publish tutorial
- [x] V4 Consume tutorial
- [x] V4 Swap tutorial
- [x] V4 Stake tutorial

### CODE REVIEW

**Comment by AnaLoznianu** [comment]
@akshay-ap I think we can improve a bit the publish flow. These are some ideas:
* "Step 1 - **Fill metadata**". Maybe **"Fill in metadata"**? Let's provide some information on the fields and also mention which ones can be edited after the asset is published.
* "Step 2 - **Fill access details"**. Maybe **"Fill in access details"?** The same, let's provide some info on the fields. Also, mention which are required and which are not.
* "Step 3 - Set pricing". the text for this section is personal and non-personal. I suggest changing this line: "Here, we are selecting dynamic pricing." to "Flow for dynamic pricing schema"
* "Step 5 - Send blockchain transactions". Here the captions seem to be connected to the next step but actually, they represent the previous step. Can we get the captions closer to the right transaction they represent? 
* "Confirmation". I suggest changing the label: "Now, after the transactions are completed, the below screen will appear." to "Once the transactions are completed, the below screen is displayed." 

I would remove the links towards a specific transaction. It might create confusion. 
"To view Create tokens and pricing transaction on testnet click [here](https://rinkeby.etherscan.io/tx/0x599a506d8429e3bd8b827f7ad2ca893ca866c8259a17d88b352c0345842e67d0)
To view Publish DDO transaction on testnet click [here](https://rinkeby.etherscan.io/tx/0x8d1a1f0d833f4e46432351d16013bb4fcdd6b05e65a2fe3e599f3a90e4d35478)
To view published the asset on Ocean Market click [here](https://v4.market.oceanprotocol.com/asset/did:op:63920bb64ad1e79bed73214e5fd574fc68c0977cdba5b7031aa98a3a83e28197)"

**Review by AnaLoznianu — CHANGES_REQUESTED** [review]

File: content/tutorials/marketplace-download-data-asset.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+### Step 2: Allow access to Ocean token(s)
+
+![consume part-3](images/marketplace/consume-2.png 'Transaction 1: Allow pool to access Ocean tokens')
+
+### Step 3: Buy datatoken by exchanging it with Ocean token(s)
+
+![consume part-4](images/marketplace/consume-3.png 'Transaction 1: Buy datatoken')
+
+### Step 4: Click buy button
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-download-data-asset.md:36]: Here the button is download

File: content/tutorials/marketplace-download-data-asset.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+The buy button will be enabled only if the wallet address has enough Ocean tokens to exchange them with 1 datatoken.
+
+![consume part-1](images/marketplace/consume-1.png 'Buy')
+
+### Step 2: Allow access to Ocean token(s)
+
+![consume part-3](images/marketplace/consume-2.png 'Transaction 1: Allow pool to access Ocean tokens')
+
+### Step 3: Buy datatoken by exchanging it with Ocean token(s)
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-download-data-asset.md:32]: Buy "a" datatoken

File: content/tutorials/marketplace-download-data-asset.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+3. Connect wallet.
+
+   ![connect wallet](images/marketplace/consume-connect-wallet.png 'Connect wallet')
+
+   In this tutorial, we will be using Rinkeby testnet.
+
+## Tutorial
+
+### Step 1: Click buy button
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-download-data-asset.md:22]: Click "on the buy" button

File: content/tutorials/marketplace-publish-data-asset.md [hunk 4]
```diff
@@ -9,62 +9,67 @@ Ocean Market provides a convenient interface for individuals as well as organiza
 
 ## Tutorial
 
-### Step 1 - Publish data asset
+### Connecting wallet and navigating to publish page
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-publish-data-asset.md:12]: navigating to "the" publish page

File: content/tutorials/marketplace-publish-data-asset.md [hunk 5]
```diff
@@ -9,62 +9,67 @@ Ocean Market provides a convenient interface for individuals as well as organiza
...
 
-4. Fill the publish form.
-
-   ![publish form part-1](images/marketplace/publish-form-1.png 'Publish form part-1')
-   ![publish form part-2](images/marketplace/publish-form-2.png 'Publish form part-2')
-   ![publish form part-3](images/marketplace/publish-form-3.png 'Publish form part-3')
+### Step 1 - Fill in metadata
 
-5. After clicking submit, approve the transactions in the wallet. Here, you can see Metamask window.
+Fill the metadata.
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-publish-data-asset.md:28]: Fill "in" the metadata

File: content/tutorials/marketplace-swap-and-stake.md [hunk 6]
```diff
@@ -1,38 +1,73 @@
 ---
 title: Swap and/or Stake Tokens
-description: 
+description:
 ---
 
 ## Swap OCEAN Tokens against datatokens
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-swap-and-stake.md:6]: Swap OCEAN "tokens" against datatokens

File: content/tutorials/marketplace-swap-and-stake.md [hunk 7]
```diff
@@ -1,38 +1,73 @@
...
 
-1. Search for the desired asset published on the [Ocean Marketplace](https://market.oceanprotocol.com/).
+### Finding an asset
 
-2. Select **Trade** option and enter the amount of Ocean tokens you want to swap. The expected amount that the account will receive will be shown with the swap fees information.
+Search for the desired asset published on the <a href="https://v4.market.oceanprotocol.com/" target="_blank">Ocean Marketplace</a>.
 
-   ![swap part-1](images/marketplace/Swap-1.png 'Select trade')
+### Step 1: Understanding the risks
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-swap-and-stake.md:12]: I would rename this step to "Agree with the risks associated".. Also with a description:
"Please read carefully the risks associated with this operation and agree to move forward"

File: content/tutorials/marketplace-swap-and-stake.md [hunk 8]
```diff
@@ -1,38 +1,73 @@
...
+
+### Step 1: Select pool tab
+
+After finding the dataset, select **`POOL`** tab and click on **`ADD LIQUIDITY`**
+
+![staking part-1](images/marketplace/Staking-1.png 'Understanding risks')
+
+### Step 2: Understanding the risks
+
+![staking part-2](images/marketplace/Staking-2.png 'Enter amount of Oceans')
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-swap-and-stake.md:52]: This image doesn't load

File: content/tutorials/marketplace-swap-and-stake.md [hunk 9]
```diff
@@ -1,38 +1,73 @@
...
+
+![swap part-5](images/marketplace/Swap-5.png 'Transaction: Swap tokens')
+
+### Viewing transaction
+
+![swap part-6](images/marketplace/Swap-6.png 'View transaction')
+
+Click <a href="https://rinkeby.etherscan.io//tx/0x2f98479adafa93eedbdff8a90a41d760a42290c961dd6aa2b9e09170665adc23" target="_blank">here</a> to view a sample transaction on Rinkeby testnet.
 
 ## Staking (Adding Liquidity)
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-swap-and-stake.md:40]: The liquidity pool tab will change a little bit. We already have a PR opened: https://github.com/oceanprotocol/market/pull/1327.
Let's please wait to have that merged and then we can update the images as well. Thanks

File: content/tutorials/marketplace-swap-and-stake.md (hunk 8 above)
Comment (akshay-ap) [reply content/tutorials/marketplace-swap-and-stake.md:56]: Might be an issue with the cache.

**Comment by AnaLoznianu** [comment]
@trentmc @alexcos20 Should we use the stake term in these tutorials? Or "Add liquidity" is more appropriate?

**Review by AnaLoznianu — APPROVED** [review]

**Review by alexcos20 — APPROVED** [review]
lgtm

**Pull request merged**